iam/README.md

```diff
@@ -102,3 +105,67 @@ roles:
 
 
 ```
+
+#### Create consumers for services
+
+This example defines a YAML file that creates roles and permissions for the DAS system `bhq`
+```yaml
+name: quotes service
+permission-prefix: bhq
+services:
+  - name: braveheart-quotes
+    repository: braveheart-quotes-service
+    allowed-consumers:
+    - clan: iam
+      service-accounts:
+        - 'iam-api@iam-staging-c2a9.iam.gserviceaccount.com'
+    - clan: quotes
+      service-accounts:
+        - 'braveheart-quotes-webclient-be@quotes-staging-ccdf.iam.gserviceaccount.com'
+  - name: braveheart-quotes-webclient-be
+    repository: braveheart-quotes-webclient-backend
+    allowed-consumers:
+    - clan: iam
+      service-accounts:
+        - 'iam-api@iam-staging-c2a9.iam.gserviceaccount.com'
+        - 'iam-das-sync-worker@iam-staging-c2a9.iam.gserviceaccount.com'
+        - 'iam-oauth-client-managment@iam-staging-c2a9.iam.gserviceaccount.com'
+permissions:
+  quote:
+    - create
+    - delete
+    - get
+    - list
+    - update
+roles:
+  - id: admin
+    name: Braveheart quotes administrator
+    desc: full access
+    permissions:
+      - 'quote.create'
+      - 'quote.delete'
+      - 'quote.get'
+      - 'quote.list'
+      - 'quote.update'
+  - id: viewer
+    name: Braveheart quotes viwer
+    desc: view access
+    permissions:
+      - 'quote.get'
+      - 'quote.list'
+
+```
+
+The consumers will be available in your das system by importing data.consumers. This dataset will contain an array named consumers with service-accounts.
+
+example policy
+
+```
+import data.consumers.consumers
```

Comment:
Not a fan of the double consumers... :)

Reply:
changed this to data.consumers.services 👍